### PR TITLE
Feature/first query

### PR DESCRIPTION
**Pull request by carolineshowscode**
## Type of Change
- [ ] JSX
- [ ] HTML
- [ ] JavaScript
- [ ] CSS
- [X] React
- [ ] React Router
- [ ] Cypress
- [X] Apollo
- [X] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

### Feature:
- installed Apollo and GraphQL for React
- wrote test query (now deleted)
- wrote first query for oneUser
- passed user props to `Form` and `Nav` components

## How has this been tested?
- [X] In Browser
- [X] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: walk-safe/src/Components/MainView/MainView.js [hunk 1]
```diff
@@ -3,17 +3,36 @@ import NavBar from '../NavBar/NavBar';
 import Header from '../Header/Header';
 import Form from '../Form/Form';
 import MapDisplay from '../MapDisplay/MapDisplay';
+import { gql, useQuery } from '@apollo/client';
+
+const GET_USER = gql`
```
Comment (pcmueller) [inline walk-safe/src/Components/MainView/MainView.js:8]: Killed it with the GraphQL syntax!

File: walk-safe/src/Components/NavBar/NavBar.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
-function NavBar() {
+function NavBar({user}) {
   return (
     <nav className='navbar'>
       <div className='welcome-container'>
-        <h2 className='welcome-msg'>Welcome, user's name!</h2>
+        <h2 className='welcome-msg'>Welcome, {user}</h2>
```
Comment (pcmueller) [inline walk-safe/src/Components/NavBar/NavBar.js:7]: Incredible interpolation

File: walk-safe/src/Components/Form/Form.js [hunk 3]
```diff
@@ -49,9 +50,9 @@ function Form() {
       <Select
         className='dropdown'
         placeholder='Select contact'
-        // defaultValue={selectedContact}
-        // onChange={setSelectedContact}
-        // options={contacts}
+        defaultValue={selectedContact}
+        onChange={setSelectedContact}
+        options={contacts}
```
Comment (pcmueller) [inline walk-safe/src/Components/Form/Form.js:55]: I'll get these dropdown options rendering as soon as possible

**Review by pcmueller — APPROVED** [review]
Amazing work, thx for figuring out all the Apollo/GraphQL boilerplate

**Review by gaelyn — APPROVED** [review]
Looks like y'all are making great progress! So exciting to get FE and BE connected. Just FYI, the contacts also have lastName available if you want it.

**Pull request merged**